src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -63,6 +64,7 @@
 public class CSharpGapicClientTransformer implements ModelToViewTransformer {
```

Comment:
Possibly it is worth considering adding a new transformer for the resource name generation, I am not sure.